src/components/Education.js

```diff
@@ -1,34 +1,131 @@
 import styles from './education.module.css';
+import { useState, Fragment } from 'react';
+import { format } from 'date-fns';
 
-function handleClick(e) {}
+export default function Education({ data, onChange, onClick }) {
+  const [isEditing, setIsEditing] = useState(false);
+
+  function handleClick() {
```

Comment:
you don't need separate function for this. Just do `setIsEditing(!setIsEditing)`;